frontend/revive-gallery-app/src/__tests__/ProductForm.test.js

```diff
@@ -1,13 +1,8 @@
 import { render, screen, fireEvent } from '@testing-library/react';
-import { MemoryRouter } from "react-router-dom";
 import ProductForm from '../components/products/ProductForm';
 
 test('renders the product form', () => {
-  render(
-    <MemoryRouter> // MemoryRouter mimics BrowserRouter or HashRouter
-        <ProductForm />
-    </MemoryRouter>
-  );
+  render(<ProductForm />);
```

Comment:
@bkarakala This does not work if the component uses useNavigate hook. Please revert it back to <MemoryRouter>